resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -105,20 +105,44 @@ public DnsNameResolverBuilder eventLoop(EventLoop eventLoop) {
         return this;
     }
 
+    /**
+     * Use {@link #datagramChannelFactory()}
+     */
+    @Deprecated
```

Comment:
Bcz this is a `protected` method in a `final` class, doesn't look like we need to clarify in javadoc what is the replacement. This method is useful only if someone sneaks into the same package. Can we simply rename it to `datagramChannelFactory()` then?